src/main/java/com/medallia/word2vec/SearcherImpl.java

```diff
@@ -79,9 +72,22 @@ private double calculateDistance(double[] otherVec, double[] vec) {
 	 * @throws UnknownWordException If word is not in the model's vocabulary
 	 */
 	private double[] getVector(String word) throws UnknownWordException {
-		if (!normalized.containsKey(word))
+		final double[] result = getVectorOrNull(word);
+		if(result == null)
 			throw new UnknownWordException(word);
-		return normalized.get(word);
+
+		return result;
+	}
+
+	private double[] getVectorOrNull(final String word) {
+    final Integer index = word2vectorOffset.get(word);
```

Comment:
There were tabs. I changed the whole file to use spaces, and I think it touched places that I didn't touch. Were there some tabs in it before?